File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/InlineQueryFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.inline.InlineQuery;

public class InlineQueryFilterValidator extends FilterValidator<InlineQuery> {

    public InlineQueryFilterValidator(CustomValidator<InlineQuery> validator) {
        super(InlineQuery.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/InlineQueryFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.inline.InlineQuery
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class InlineQueryFilterValidator(validator: Validator<InlineQuery>) : FilterValidator<InlineQuery>(
    InlineQuery::class, validator)

